### PR TITLE
Adding Python Local Bucketing SDK 

### PR DESCRIPTION
**Pull request by chris-hoefgen**
- updated code samples
- Added content for the Local Bucketing SDK

### CODE REVIEW

File: docs/sdk/server-side-sdks/python/python-gettingstarted.md [hunk 1]
```diff
@@ -8,22 +8,44 @@ sidebar_custom_props: {icon: rocket}
...
+
+```python
+    from devcycle_python_sdk import DevCycleLocalClient, DevCycleLocalOptions
+    from devcycle_python_sdk.models.user import DevCycleUser
+    
+    # Create an options object to do custom configurations, or use the defaults
+    options = DevCycleLocalOptions()
+    
+    # create an instance of the DevCycleCloudClient class
+    client = DevCycleLocalClient('YOUR_DVC_SERVER_SDK_KEY', options)
```
Comment (jonathannorris) [inline docs/sdk/server-side-sdks/python/python-gettingstarted.md:23]:
```suggestion
    client = DevCycleLocalClient('YOUR_DEVCYCLE_SERVER_SDK_KEY', options)
```

File: docs/sdk/server-side-sdks/python/python-gettingstarted.md [hunk 2]
```diff
@@ -8,22 +8,44 @@ sidebar_custom_props: {icon: rocket}
...
     from devcycle_python_sdk import DevCycleCloudClient, DevCycleCloudOptions
-    from devcycle_python_sdk.models.user import User
+    from devcycle_python_sdk.models.user import DevCycleUser
     
     # Create an options object and enable storing user data in EdgeDB
     options = DevCycleCloudOptions(enable_edge_db=True)
     
     # create an instance of the DevCycleCloudClient class
-    dvc = DevCycleCloudClient('YOUR_DVC_SERVER_SDK_KEY', options)
+    client = DevCycleCloudClient('YOUR_DVC_SERVER_SDK_KEY', options)
```
Comment (jonathannorris) [inline docs/sdk/server-side-sdks/python/python-gettingstarted.md:45]:
```suggestion
    client = DevCycleCloudClient('YOUR_DEVCYCLE_SERVER_SDK_KEY', options)
```

File: docs/sdk/server-side-sdks/python/python-usage.md [hunk 3]
```diff
@@ -37,67 +37,86 @@ to be fetched from DevCycle's CDN.
...
 If you would like to get the full Variable you can use `variable()` instead. This contains fields such as:
 `key`, `value`, `type`, `defaultValue`, `isDefaulted`.
 
 ## Getting All Variables
 
-To get all variables, use the `all_variables()` method to retrieve a dict with all variables that the user receives.
+Use the `all_variables()` method to retrieve a dictionary with all the segmented variables for the user.
 
 ```python
     try:
```
Comment (jonathannorris) [inline docs/sdk/server-side-sdks/python/python-usage.md:57]: Should all these examples be indented? It looks a little bit off on the page: 
image

File: docs/sdk/server-side-sdks/python/python-gettingstarted.md (hunk 1 above)
Comment (chris-hoefgen) [reply docs/sdk/server-side-sdks/python/python-gettingstarted.md]: personally, no i don't think so.  seems like we have a habit of sprinkling that in here and there but aren't consistent.  I'll remove it.

File: docs/sdk/server-side-sdks/python/python-usage.md (hunk 3 above)
Comment (chris-hoefgen) [reply docs/sdk/server-side-sdks/python/python-usage.md]: agreed. i'll remove the indent.  I think it was there already and I just didn't think to fix it.

**Review by jonathannorris — APPROVED** [review]

**Pull request merged**